skfusion/fusion/base/fusion_graph.py

```diff
@@ -39,33 +39,49 @@ def __getitem__(self, key):
     def __setitem__(self, key, value):
         self.adjacency_matrix[key] = value
 
-    def draw_graphviz(self, filename):
-        """Draw the data fusion graph and save it to a file (SVG).
+    def draw_graphviz(self, *args, **kwargs):
+        """Draw the data fusion graph using PyGraphviz and save it to a file.
 
         Parameters
         ----------
-        filename :
+        verbose_edges : bool, optional
+            Annote edge labels with relations' matrix shape information.
+
+        *args, **kwargs : optional
+            Passed to `pygraphviz.AGraph.draw()` method.
         """
+        verbose_edges = kwargs.pop('verbose_edges', False)
         import pygraphviz as pgv
-        fus_graph = pgv.AGraph(strict=False, directed=True)
-        # object types
-        for ot in self.object_types:
-            fus_graph.add_node(ot.name)
-        # relations
-        ot2count = defaultdict(int)
-        for relation in self.relations:
-            ot1 = relation.row_type
-            ot2 = relation.col_type
-            ot2count[ot1, ot2] += 1
-            if ot1 != ot2:
-                label = '<<b>R</b><SUB>%s,%s</SUB><SUP>%d</SUP>' \
-                        '<br/>>' % (ot1.name, ot2.name, ot2count[ot1, ot2])
-                fus_graph.add_edge(ot1.name, ot2.name, text=label)
+        G = pgv.AGraph(strict=False, directed=True)
+        G.node_attr['fontsize'] = 11
+        G.edge_attr['fontsize'] = 9
+        G.node_attr['fontname'] = \
+        G.edge_attr['fontname'] = 'sans-serif'
+
+        G.add_nodes_from(o.name for o in self.object_types)
+
+        relations = defaultdict(list)
+        for rel in self.relations:
+            relations[(rel.row_type, rel.col_type)].append(rel)
+        import numpy as np
+        meanweight = np.mean([r.data.shape[0] for rels in relations.values() for r in rels])
+        for (ot1, ot2), rels in relations.items():
+            if ot1 == ot2:
+                label = '<b>&Theta;</b><font point-size="6">%s</font>' % ot1.name
```

Comment:
I know about the trouble with super/sub-scripts. It exists on OS X as well. Your solution is fine.